### DISCUSSION TITLE
Pterodactyl panel support?

### DISCUSSION THREAD

**Discussion by l0g4n1337** [jef/streetmerchant#3100] [Ideas and feature requests]
Hi, I use pterdactyl panel, unfortunately I can't install streetmerchant because I have problems with chromium.
It would be nice if something could be done about this to allow an install on pterodactyl as well.

**Comment by Mr-Tech-13** [comment]
https://github.com/jef/streetmerchant/wiki/Installation:-Linux:-Debian-based

This may help

**Reply by l0g4n1337** [reply]
pterodactyl use egg systems so ... its a nightmare.
You cant use command.
So for this we need an expert of ptero.
